### PR TITLE
fixes: multiline code snippet buttons doesn't work properly

### PR DESCRIPTION
**Pull request by sahil143**
fixes: https://issues.redhat.com/browse/ODC-5896

This PR fixes the issue of the multiline/single line code snippets where the event listeners were not attached which made both `copy to clipboard` and `execute in terminal` buttons unresponsive.

Kapture 2021-06-01 at 22 20 01

### CODE REVIEW

File: frontend/packages/console-shared/src/components/markdown-extensions/MarkdownCopyClipboard.tsx [hunk 1]
```diff
@@ -62,6 +62,19 @@ const MarkdownCopyClipboard: React.FC<MarkdownCopyClipboardProps> = ({
...
+   * because we use `dangerouslySetInnerHTML` to set the markdown markup to a div.
+   * So while react updates the dom with latest of markdown markup with `dangerouslySetInnerHTML`
+   * this component mounts and access the old copy of dom in which there are no elements present
+   * with querySelector.
+   * So using forceRender to rerender the component and access the `document` in which markdown markup is present.
+   */
+  React.useEffect(() => {
+    forceRender();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (christianvogt) [inline frontend/packages/console-shared/src/components/markdown-extensions/MarkdownCopyClipboard.tsx:77]: You should update `InlineMarkdownView` instead to delay the rendering of the extensions by one render cycle.

**Comment by christianvogt** [comment]
Sent some comments directly to @sahil143 on slack.

**Comment by invincibleJai** [comment]
/lgtm

verified this PR with cluster bot and the changes works as expected.

**Pull request merged**